.travis.yml

```diff
@@ -0,0 +1,18 @@
+language: python
+
+python:
+  - 3.6
+
+dist: xenial
+sudo: false
+
+cache: pip
+
+install:
+  - pip install --upgrade pycodestyle
```

Comment:
@shm0o 
True, but I'm overriding this default by setting the value for the `install` key,

see https://docs.travis-ci.com/user/languages/python/#pip

> By default Travis CI uses `pip` to manage python dependencies. If you have a `requirements.txt` file, Travis CI runs `pip install -r requirements.txt` during the `install` phase of the build.
> 
> You can manually override this default `install` phase, for example: